tools/shared/pragma.c

```diff
@@ -1295,8 +1297,16 @@ do_sw(void)
      * mark routine or all routines as not-to-be-extracted, and therefore
      * not to be inlined
      */
+    bclr(DIR_OFFSET(currdir, x[191]), 0x2);
     bset(DIR_OFFSET(currdir, x[14]), 8);
     break;
+  case SW_FORCEINLINE:
+    if (no_specified)
+      break;
+    else
+      bclr(DIR_OFFSET(currdir, x[14]), 0x8);
+      bset(DIR_OFFSET(currdir, x[191]), 0x2);
```

Comment:
NIT: This seems ambiguous(2 statements following `else` without explicit braces) ?
WDYT?

Reply:
Definitely needs to be addressed.

Reply:
That's a mistake on my part. Apologies, thanks for finding it.
Fixed.